### PR TITLE
Students

### PR DESCRIPTION
**Pull request by komalahire**
I have done an edit profile.

### CODE REVIEW

File: src/components/ getAllStudentsDetails.js [hunk 1]
```diff
@@ -38,169 +39,201 @@ export class GetAllStudentsDetails extends Component {
...
         });
       })
-      .catch((error) => {
+      .catch(error => {
         console.log(error, "error");
       });
   }
 
+  submit = data => {
+    console.log(data);
```
Comment (himanshu0128) [inline src/components/ getAllStudentsDetails.js]: remove this before pushing code to github

File: src/components/ getAllStudentsDetails.js [hunk 2]
```diff
@@ -38,169 +39,201 @@ export class GetAllStudentsDetails extends Component {
...
+      student: data,
+      showForm: true
+    });
+   
+  };
+
   render() {
-    console.log(this.state.studentsDetails);
     const { classes } = this.props;
+    console.log(this.state.showForm,"showForm")
```
Comment (himanshu0128) [inline src/components/ getAllStudentsDetails.js]: no console statements

File: src/components/addressForm.js [hunk 3]
```diff
@@ -304,6 +327,9 @@ export class addressForm extends Component {
       { label: "Uttarakhand" },
       { label: "West Bengal" },
     ];
+
+    console.log(this.state. parents_name);
```
Comment (himanshu0128) [inline src/components/addressForm.js]: remove this

File: src/components/EditProfile.js [hunk 4]
```diff
@@ -0,0 +1,2 @@
+
```
Comment (himanshu0128) [inline src/components/EditProfile.js]: Delete this file. Please look at the changes before pushing code to GitHub. Use the Git diff command and git status command to see what are you pushing.

**Review by himanshu0128 — CHANGES_REQUESTED** [review]

File: src/components/ getAllStudentsDetails.js (hunk 1 above)
Comment (komalahire) [reply src/components/ getAllStudentsDetails.js]: I have resolved

File: src/components/ getAllStudentsDetails.js (hunk 2 above)
Comment (komalahire) [reply src/components/ getAllStudentsDetails.js]: resolved

File: src/components/addressForm.js (hunk 3 above)
Comment (komalahire) [reply src/components/addressForm.js]: I have resolved

**Pull request merged**